formats/polus-ome-zarr-converter-plugin/src/main.py

```diff
@@ -34,30 +33,41 @@ def image_to_zarr(inp_image: Path, out_dir: Path) -> None:
                 # Loop through channels
                 for c in range(br.C):
 
-                    extension = "".join(
-                        [
-                            suffix
-                            for suffix in inp_image.suffixes[-2:]
-                            if len(suffix) < 5
-                        ]
-                    )
+                    if fileExtension == ".ome.tif":
+
+                        extension = "".join(
+                            [
+                                suffix
+                                for suffix in inp_image.suffixes[-2:]
+                                if len(suffix) < 6
+                            ]
+                        )
```

Comment:
Don't use an if block here. You are correct to change this to the length check to `< 6` because I forgot in the template code to account for `.`.

Reply:
I removed if block. suffix length is set to be < than 6 which works for both file format